mod-di-converter-storage-server/src/main/resources/templates/db_scripts/associations-migration-gen-table/associations_view.sql

```diff
@@ -0,0 +1,48 @@
+-- Custom script to create associations_view view. Changes in this file will not result in an update of the view.
+-- To change the associations_view view, update this script and copy it to the appropriate scripts.snippet field of the schema.json
+
+CREATE OR REPLACE VIEW associations_view
+  AS
+    SELECT association_id, master_id, masterwrapperid, master, master_type, detail_id, detailwrapperid, detail_type, detail_order, detail, react_to, job_profile_id
+    FROM get_profile_association_snapshot('job_profiles', 'JOB_PROFILE', 'match_profiles', 'MATCH_PROFILE')
+      UNION ALL
+    SELECT association_id, master_id, masterwrapperid, master, master_type, detail_id, detailwrapperid, detail_type, detail_order, detail, react_to, job_profile_id
+    FROM get_profile_association_snapshot('job_profiles', 'JOB_PROFILE', 'action_profiles', 'ACTION_PROFILE')
+      UNION ALL
+    SELECT association_id, master_id, masterwrapperid, master, master_type, detail_id, detailwrapperid, detail_type, detail_order, detail, react_to, job_profile_id
+    FROM get_profile_association_snapshot('match_profiles', 'MATCH_PROFILE', 'match_profiles', 'MATCH_PROFILE')
+     UNION ALL
+    SELECT association_id, master_id, masterwrapperid, master, master_type, detail_id, detailwrapperid, detail_type, detail_order, detail, react_to, job_profile_id
+    FROM get_profile_association_snapshot('match_profiles','MATCH_PROFILE', 'action_profiles', 'ACTION_PROFILE')
+     UNION ALL
+    SELECT association_id, master_id, masterwrapperid, master, master_type, detail_id, detailwrapperid, detail_type, detail_order, detail, react_to, job_profile_id
+    FROM get_profile_association_snapshot('action_profiles', 'ACTION_PROFILE', 'match_profiles', 'MATCH_PROFILE')
+      UNION ALL
+    SELECT association_id, master_id, masterwrapperid, master, master_type, detail_id, detailwrapperid, detail_type, detail_order, detail, react_to, job_profile_id
+    FROM get_profile_association_snapshot('action_profiles', 'ACTION_PROFILE', 'action_profiles', 'ACTION_PROFILE')
```

Comment:
please, remove these calls of function for non-existing types of associations

Reply:
thank you, done!